### ISSUE TITLE
Hackney version is broken with hex 

### ISSUE THREAD

**Issue by timmak** [benoitc/couchbeam#159]
Current version 1.6.2 which is broken on hex #157 seems to provide a fix for it maybe it need to be a the specific version though

**Comment by dc0d** [comment]
Seems `hackney` version get relaxed. Instead of the strict version `"1.6.2"`, `"~> 1.6.2"` could be used. The other day I was using `httpoision` and their `hackney` version was `"~> 1.6.3"` and caused a conflict.

(As an Elixir Noob, it took me some hours to make (/force) things to work! :D)

**Comment by chrisortman** [comment]
I worked around this with

```
     {:hackney, "~> 1.6", override: true},
     {:couchbeam, "~> 1.4"}
```

In case someone else comes looking for an answer in the interim

**Comment by benoitc** [comment]
fixed by d7c3f476287467291c065d3ba4c58bdf8d990923 now only building using rebar3
